NBitcoin/BIP32/KeyPath.cs

```diff
@@ -40,6 +40,23 @@ public KeyPath(string path)
 				.ToArray();
 		}
 
+		public static KeyPath FromBytes(byte[] data)
+		{
+			if (data.Length % 4 != 0)

```

Comment:
It is a common practice for public methods to check argument for NULL to prevent NREs. Q: should a zero-length byte array be allowed here?

Reply:
I think it should be allowed to represent path for the master key. I will add test